themes/tiddlywiki/vanilla/reset.tid

```diff
@@ -1,5 +1,7 @@
 title: $:/themes/tiddlywiki/vanilla/reset
 type: text/css
+tags: $:/tags/Stylesheet $:/tags/Stylesheet/Static
+list-before: $:/themes/tiddlywiki/vanilla/base
```

Comment:
hmmm, IMO that's not possible. If a user modifies the sort order by hand this shadow tiddler is converted to a system-tiddler and the `list-before` field is removed. -> That can lead to future problems for the user, if we change the `reset.css` again, because it is **no shadow** anymore

We will probably need to rename the tiddler to `$:/themes/tiddlywiki/vanilla/_reset`, so it will be automatically listed in front. 

@Jermolene -- The **problem** is, that renaming is not 100% backwards compatible